scripts/composer.sh

```diff
@@ -7,6 +7,9 @@ curl -sS https://getcomposer.org/installer | php
 echo "Moving composer to make it globally accessable"
 mv composer.phar /usr/bin/composer
 
+composer config -g notify-on-install false
```

Comment:
How comes? My view is this is a dev environment where the installation is throw away. Installations should probably only be notified from near live environments and not throw away test machines.